src/blog/2024/08/using-mqtt-sparkplugb-with-node-red.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Using MQTT Sparkplug B with Node-RED
+subtitle: A Practical Guide to integrating MQTT Sparkplug B with Node-RED
+description: Explore how MQTT Sparkplug B enhances MQTT with standardized data formats, making integration easier. Learn how to set up Sparkplug B in Node-RED for efficient data management in industrial environments.
+date: 2024-08-21
+authors: ["sumit-shinde"]
+image: 
+tags:
+   - post
+   - nodered
+   - mqtt sparkplug b with node-red
+---
+
+Connected devices are creating a lot of data, but without a standard format, managing this data can be tricky. MQTT helps with messaging, but it can be inconsistent. MQTT Sparkplug B solves this problem by providing a clear, standardized format for data. In this guide, we’ll show you how to use MQTT Sparkplug B with Node-RED to make managing your device data easier and more organized.
```

Comment:
```suggestion
Connected devices are creating a lot of data but without a standard format, managing this data can be tricky. MQTT helps with messaging, though MQTT is flexible and doesn't standardize message formats. MQTT Sparkplug B solves this problem by providing a clear, standardized format for data. In this guide, we’ll show you how to use MQTT Sparkplug B with Node-RED to make managing your device data easier and more organized.
```

Reply:
NB: A counter suggestion was offered in my review here: https://github.com/FlowFuse/website/pull/2478#discussion_r1724909709

Reply:
outdated